lesson_23_selenide/src/test/java/pageobject_model/ProductPage.java

```diff
@@ -0,0 +1,59 @@
+package pageobject_model;
+
+import com.codeborne.selenide.Condition;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class ProductPage {
+    static By buyButton = By.id("button-cart");
+    static By cartModal = By.id("fm-cart-modal");
+    static By closeCartButton = By.cssSelector("#fm-popup-cart > div > div > div.modal-header > button");
+    static By wishListButton = By.linkText("В закладки");
+    static By alertToast = By.className("alert-block");
+    static By alertToastText = By.className("fm-alert-text");
```

Comment:
the same field as for  HeaderMenu, logic could be moved into some base class or page-element class